### DISCUSSION TITLE
Why is traefik disabled by default?

### DISCUSSION THREAD

**Discussion by UntouchedWagons** [techno-tim/k3s-ansible#392] [Questions] [answered]
In the [sample all.yaml](https://github.com/techno-tim/k3s-ansible/blob/master/inventory/sample/group_vars/all.yml#L39) traefik is flagged as disabled and in your [instructions](https://technotim.live/posts/k3s-etcd-ansible/) it says "It’s best to start using these args, and optionally include traefik if you want it installed with k3s however I would recommend installing it later with helm".

I do install traefik later with helm but I don't see the advantage of using helm to install traefik (or using helm at all to be honest) rather than have it installed by default. What would I need to change to use built-in traefik?

**Comment by timothystewart6** [comment, marked as answer]
Hey!  Sorry for the delay.  The reason the traefik is disabled in the initial install is because I feel there are better ways to install it vs the way k3s handles it.  Also, it makes upgrades and even uninstalling it easier when you install it yourself.
